verify

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-mocha_multi="./mocha-multi.js"
+mocha_multi="mocha-multi.js"
```

Comment:
Require for tests to pass with `mocha` v2, v3, and v4.  I didn't investigate why this change is required on older versions of `mocha`, since it seems safe enough.  Also works for v5 and v6.